app/src/main/java/dominando/android/livros/BookListFragment.kt

```diff
@@ -89,16 +103,15 @@ class BookListFragment : BaseFragment() {
         lifecycle.addObserver(viewModel)
     }
 
-    private fun updateList(books: List<Book>?) {
-        books?.let {
-            val columns =
-                    if (resources.configuration.orientation == Configuration.ORIENTATION_PORTRAIT) 1
-                    else 2
-            rvBooks.layoutManager = GridLayoutManager(requireContext(), columns)
-            rvBooks.adapter = BookAdapter(books) { book ->
-                router.showBookDetails(book)
-            }
-            attachSwipeToRecyclerView()
+    private fun initRecyclerView(recyclerView: RecyclerView) {
```

Comment:
Why pass the `RecyclerView` as parameter once it is a member field?

Reply:
Fixed up